tests/components/srp_energy/test_sensor.py

```diff
@@ -0,0 +1,130 @@
+"""Tests for the srp_energy sensor platform."""
+from homeassistant.components.srp_energy.const import (
+    ATTRIBUTION,
+    DEFAULT_NAME,
+    DOMAIN,
+    ICON,
+    SENSOR_NAME,
+    SENSOR_TYPE,
+)
+from homeassistant.components.srp_energy.sensor import SrpEntity, async_setup_entry
+from homeassistant.const import ATTR_ATTRIBUTION, ENERGY_KILO_WATT_HOUR
+
+from tests.async_mock import MagicMock
+
+# async def test_sensor(hass):
```

Comment:
remove commented code...

Reply:
@dgomes I removed the comments and the deepcopy.